inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarUtils.java

```diff
@@ -273,7 +273,7 @@ public static void createPartitionedTopic(RestTemplate restTemplate, PulsarClust
      * @return pulsar internal stat info of partitioned topic
      * @throws Exception any exception if occurred
      */
-    public static JsonObject getPulsarStatsPartitionedTopics(RestTemplate restTemplate,
+    public static JsonObject getStatsPartitionedTopics(RestTemplate restTemplate,
```

Comment:
First of all, `partitioned-stats` and `partitioned-statsInternal` are two different REST APIs for the pulsar.
The method name `getStatsPartitionedTopics` is not match with the URL `clusterInfo.getAdminUrl() + QUERY_PERSISTENT_PATH + "/" + topicPath + "/partitioned-internalStats"`